plugins/slack/alerta_slack.py

```diff
@@ -12,6 +12,7 @@
 SLACK_WEBHOOK_URL = os.environ.get('SLACK_WEBHOOK_URL') or app.config['SLACK_WEBHOOK_URL']
 SLACK_ATTACHMENTS = True if os.environ.get('SLACK_ATTACHMENTS', 'False') == 'True' else app.config.get('SLACK_ATTACHMENTS', False)
 SLACK_CHANNEL = os.environ.get('SLACK_CHANNEL') or app.config.get('SLACK_CHANNEL', '')
+SLACK_CHANNEL_ENV_MAP = os.environ.get('SLACK_CHANNEL_ENV_MAP') or app.config.get('SLACK_CHANNEL_ENV_MAP', None)
```

Comment:
Instead of `None` you could put `dict()` and then the below becomes simply ...
```
channel = SLACK_CHANNEL_ENV_MAP.get(alert.environment, SLACK_CHANNEL)
```
ie. you don't need to wrap line 59 in an `if` clause.